app/models/user.rb

```diff
@@ -1,8 +1,9 @@
 class User < ApplicationRecord
   before_save { email.downcase! }
-  validates :name, presence: true, length: { maximum: 50 }
-  VALID_EMAIL_REGEX = Settings.valid_email_regex
-  validates :email, presence: true, length: { maximum: 255 }, format: { with: VALID_EMAIL_REGEX }, uniqueness: { case_sensitive: false }
+  validates :name, presence: true, length: { maximum: Settings.length_name }
+  VALID_EMAIL_REGEX = Settings.email_regex
+  validates :email, presence: true, length: { maximum: Settings.length_email },
+  format: { with: VALID_EMAIL_REGEX }, uniqueness: { case_sensitive: false }
   has_secure_password
   validates :password, presence: true, length: { minimum: 6 }
```

Comment:
For this file: 
1. Please move constant to settings.yml
2. Remove white space after the start and before the end of curly brace